Automattic/jetpack
# VideoPress: Loading mode for `VideoRow`

Closes #26796 

#### Changes proposed in this Pull Request:

* Add `disabled` state for `Checkbox`.
* Add `blankIconSize` prop in `VideoThumbnail`.
* Support `loading` in `VideoRow`.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `VideoPress`.
* Navigate to `Dashboard`.
* Change view for list.
* Paginate or search for something.
* You should see it in loading state.

#### Demo


![Screen Shot 2022-10-13 at 17 12 08](https://user-images.githubusercontent.com/1663717/195699838-e58e979e-15ad-4a10-8a4b-e521cdbb5b07.png)
